Jenkinsfile

```
@@ -1,14 +1,13 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@test/cdn') _
```

TODO: to be reverted once the library has been released
```suggestion
@Library('apm@current') _
```